### PR TITLE
feat: dagger do should list all nested actions

### PR DESCRIPTION
**Pull request by morlay**
Nested sub actions should listed too

```
Available Actions:
  go                     
  go source              
  go ship                
  go ship pushx          Push all images as multi-arch images
  go ship push           
  go ship push x         Merge arch suffix image in to mutli-arch images
  go ship push amd64     Push <arch> suffix image
  go ship push arm64     Push <arch> suffix image
  go ship load           Load built image to local docker
  go ship load amd64     
  go ship load arm64     
  go build               
  go build linux/amd64   Go build binary for special platform
  go build linux/arm64   Go build binary for special platform
  go build darwin/amd64  Go build binary for special platform
  go build darwin/arm64  Go build binary for special platform
  go devkit              
  go devkit load         Load go build env image to local docker
  go devkit load amd64   
  go devkit load arm64   
  go archive             Archive all built binaries into local (need to define client: `filesytem: "x": write: contents: actions.go.archive.output`)       
```

### CODE REVIEW

**Comment by morlay** [comment]
But missing `.Documentation`  when in `#Action` from `cue.mod/pkg`

File: cmd/dagger/cmd/do.go [hunk 1]
```diff
@@ -349,13 +350,24 @@ func printActions(p *plan.Plan, action *plan.Action, w io.Writer, target cue.Pat
...
+		sort.Slice(action.Children, func(i, j int) bool {
+			return action.Children[i].Name < action.Children[j].Name
+		})
+
+		for _, a := range action.Children {
+			if !a.Hidden {
+				lineParts := []string{"", strings.Join(append(parents, a.Name), " "), a.Documentation}
+				fmt.Fprintln(tw, strings.Join(lineParts, "\t\t"))
+				printAction(a, append(parents, a.Name)...)
+			}
```
Comment (TomChv) [inline cmd/dagger/cmd/do.go:365]: Nit: You could just put this in a function instead of a variable

Comment (TomChv) [inline cmd/dagger/cmd/do.go:365]: Nit: You could just put this in a function instead of a variable

**Review by TomChv — COMMENTED** [review]
Thanks for your contribution :rocket:

I think it's a nice improvement, but I think it must be possible to control the depth of research.
For instance, it must be possible to only see the first layer of commands, or first and second, etc...

Maybe that can be set with a flag?

For now, you can wait for team's feedback and then apply comments, do not put more effort until team give opinion :D

**Comment by morlay** [comment]
> Maybe that can be set with a flag?

Or just follow the depth of matched `target` path?

```cue
dagger do go build help
```

`go build` found, so just list sub actions of `go build`

**Comment by TomChv** [comment]
@morlay Is this PR still accurate or we can close it?

**Pull request closed without merging**